website/versions.json

```
@@ -1,5 +1,6 @@
 [
-  "2.11.0",
+  "2.12.0",
+  "2.12.0",
```

Hmm, something went wrong doc has been upgraded.

```suggestion
  "2.12.0",
  "2.11.0",
```

---

WTF. Ah. ok.